src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -909,6 +918,20 @@ FixedwingPositionControl::control_auto(const hrt_abstime &now, const Vector2d &c
 		publishOrbitStatus(current_sp);
 	}
 
+	// update lateral guidance timesteps for slewrates
+	if (_param_fw_use_npfg.get()) {
+		_npfg.setDt(dt);
```

Comment:
Not an issue here, but isn't the setting of Dt missing still for control_auto_descend(), control_auto_fixed_bank_alt_hold() and control_manual_position()? Shouldn't we just do it outside, in the main Run() function? Then we also just call update_position_control_mode_timestep() right? Or am I missing something?

Reply:
good catch - `control_manual_position()` should also have the dt update for npfg and l1. `descend` and `fixed_bank_alt_hold` dont use the guidance laws, so no need there. 

@dagar you were the one who originally moved the dt calculation into each resp. control method. if we are comparing these all to the `_local_pos.timestamp` anyway, there shouldn't be any issue to update the slewrates in `run()`, right?

Reply:
FYI - centralized the dt calculation in a new PR #19595

Reply:
alright :+1: 
Do you still want to add it to control_manual_position() already for this PR?